exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/AsyncPageReader.java

```diff
@@ -462,7 +431,10 @@ public Void call() throws IOException {
           readStatus.setDiskScanTime(timeToRead);
           assert (totalValuesRead <= totalValuesCount);
         }
-        synchronized (queue) {
+        // FindBugs reports this is a possible bug, but it is not. You do need the synchronized block
```

Comment:
I'd intended to suppress the warning initially, but then got rid of it by using a different object to synchronize on. Fixed the comment to not refer to FindBugs though.